From 237007a5b9e5130162776add8ca25f447a885b83 Mon Sep 17 00:00:00 2001
From: Nicholas Piggin <npiggin@gmail.com>
Date: Wed, 21 Feb 2018 05:08:28 +1000
Subject: [PATCH 05/13] powerpc: Remove unused CPU_FTR_ARCH_201

References: bsc#1055117
Patch-mainline: v4.17-rc1
Git-commit: 3735eb850e6c3e7472329fea368e7d030a4d89dd

The last usage was removed in c17b98cf6028 ("KVM: PPC: Book3S HV:
Remove code for PPC970 processors") (Dec 2014).

Signed-off-by: Nicholas Piggin <npiggin@gmail.com>
Signed-off-by: Michael Ellerman <mpe@ellerman.id.au>
Acked-by: Michal Suchanek <msuchanek@suse.de>
---
 arch/powerpc/include/asm/cputable.h | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/arch/powerpc/include/asm/cputable.h b/arch/powerpc/include/asm/cputable.h
index efece4ab178c..0f2fff7c0dcd 100644
--- a/arch/powerpc/include/asm/cputable.h
+++ b/arch/powerpc/include/asm/cputable.h
@@ -182,7 +182,6 @@ static inline void cpu_feature_keys_init(void) { }
 
 #define CPU_FTR_REAL_LE			LONG_ASM_CONST(0x0000000000001000)
 #define CPU_FTR_HVMODE			LONG_ASM_CONST(0x0000000000002000)
-#define CPU_FTR_ARCH_201		LONG_ASM_CONST(0x0000000000004000)
 #define CPU_FTR_ARCH_206		LONG_ASM_CONST(0x0000000000008000)
 #define CPU_FTR_ARCH_207S		LONG_ASM_CONST(0x0000000000010000)
 #define CPU_FTR_ARCH_300		LONG_ASM_CONST(0x0000000000020000)
@@ -418,7 +417,7 @@ static inline void cpu_feature_keys_init(void) { }
 	    CPU_FTR_MMCRA | CPU_FTR_CP_USE_DCBTZ | \
 	    CPU_FTR_STCX_CHECKS_ADDRESS)
 #define CPU_FTRS_PPC970	(CPU_FTR_LWSYNC | \
-	    CPU_FTR_PPCAS_ARCH_V2 | CPU_FTR_CTRL | CPU_FTR_ARCH_201 | \
+	    CPU_FTR_PPCAS_ARCH_V2 | CPU_FTR_CTRL | \
 	    CPU_FTR_ALTIVEC_COMP | CPU_FTR_CAN_NAP | CPU_FTR_MMCRA | \
 	    CPU_FTR_CP_USE_DCBTZ | CPU_FTR_STCX_CHECKS_ADDRESS | \
 	    CPU_FTR_HVMODE | CPU_FTR_DABRX)
-- 
2.13.6

